response.go

```diff
@@ -204,19 +199,15 @@ func unpackResponseBody(decoder *Decoder, resp interface{}) error {
 			}
 		}
 
-		// If the return value is nil,
-		// we should consider it normal
-		if rsp == nil {
-			return nil
-		}
+		response.RspObj = rsp
```

Comment:
@LaurenceLiZhixin @AlexStocks   just assign the decoded object to the response，I think this is the biggest change, which should be tested in dubbo-go.

Reply:
Got it.